rails_tutorial/app/helpers/sessions_helper.rb

```diff
@@ -4,14 +4,35 @@ def log_in user
   end
 
   def current_user
-    @current_user ||= User.find_by id: session[:user_id]
+    if user_id = session[:user_id]
+      @current_user ||= User.find_by(id: user_id) if session[:user_id]
```

Comment:
```suggestion
      @current_user ||= User.find_by id: user_id
```

CHỗ này bỏ cái phần đấy đi đc ko nhỉ